Source/DafnyDriver/DafnyDoc.cs

```diff
@@ -0,0 +1,1142 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.IO;
+using System.Linq;
+using System.Diagnostics;
+using System.Text.RegularExpressions;
+using System.Text;
+
+using static Microsoft.Dafny.DafnyDocHtml;
+
+namespace Microsoft.Dafny;
+
+/* 
+TODO:
```

Comment:
Please put this TODO in a separate issue. That will save you 4% of the size of the file.

Reply:
OK, time to remove it then.

Reply:
Removed.